cinema_project/cinema_project/settings.py

```diff
@@ -110,11 +110,11 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'EET'
 
 USE_I18N = True
 
-USE_TZ = True
+USE_TZ = False
```

Comment:
I recommend enabling this.

Reply:
![image](https://user-images.githubusercontent.com/65079954/184302031-cafa96ec-d27c-45d9-b1ff-2e3d3a69cbc0.png)

Reply:
With our without USE TZ True or False, the cancellation of reservations work, but the string representation is displayed incorrectly

Reply:
I will leave use tz false so I can see information clearly in ADMIN, but created a task in trello to fix this later on